maven-core/src/main/java/org/apache/maven/project/artifact/ActiveProjectArtifact.java

```diff
@@ -120,6 +127,12 @@ public void setFile( File destination )
         project.getArtifact().setFile( destination );
     }
 
+    /** {@inheritDoc} */
+    public void setPath( Path path )
+    {
+        this.setFile( path == null ? null : path.toFile() );
+    }
+
```

Comment:
did you override default implementation from interface?

Reply:
Yes, implicitly, as this class extends `DefaultArtifact`.